py-polars/src/polars/dataframe/frame.py

```diff
@@ -4162,7 +4162,7 @@ def write_parquet(
             return
 
         target: str | Path | IO[bytes] | _SinkDirectory = file
-        engine: EngineType = "in-memory"
+        engine: EngineType = "streaming"
```

Comment:
Is this the fix? If so, shouldn't we fix something in the in-memory path as well?